GitUI/Theme/ThemePersistence.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Windows.Forms;
+
+namespace GitUI
+{
+    public static class ThemePersistence
```

Comment:
Does it have to be `static`?
Statics increase coupling and make testing story very difficult.

This Q goes to all other new API

Reply:
ok